package.json

```diff
@@ -30,11 +30,13 @@
     "@sentry/integrations": "^7.50.0",
     "@sentry/vue": "^7.50.0",
     "axios": "^0.27.2",
-    "core-js": "^3.25.2",
+    "core-js": "^3.8.3",
```

Comment:
This is a step backwards but that's OK -- the next ticket will take this to ^3.30.1.